File: Service/Implement/DanhMucNganhNgheService.cs
namespace Service.Implement
{
    public class DanhMucNganhNgheService : BaseService<DanhMucNganhNghe, IDanhMucNganhNgheRepository>
    , IDanhMucNganhNgheService
    {
    private readonly IDanhMucNganhNgheRepository _DanhMucNganhNgheRepository;
    public DanhMucNganhNgheService(IDanhMucNganhNgheRepository DanhMucNganhNgheRepository) : base(DanhMucNganhNgheRepository)
    {
    _DanhMucNganhNgheRepository = DanhMucNganhNgheRepository;
    }
    }
    }

File: CMS/src/app/shared/DanhMucNganhNghe.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucNganhNghe } from 'src/app/shared/DanhMucNganhNghe.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucNganhNgheService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucNganhNghe";
    }
}

